Pipfile

```diff
@@ -0,0 +1,24 @@
+[[source]]
```

Comment:
*pretend I'm commenting the whole file*

I think `Pipfile.lock` should be commited because this is an application not a library, so the exact versions matter a lot.

Also, consider [Poetry](https://python-poetry.org/), I have [a branch on hathor-core](https://github.com/HathorNetwork/hathor-core/tree/chore/replace-pipenv-with-poetry) with it, and it's very similar to setup. If we want to move away from Pipenv, it's a good thing to test the alternatives on smaller deploys.

Reply:
I'll do it. Thanks!

Reply:
I kept Pipfile for now. But I'll open an issue to migrate to Poetry soon. Ok?

Reply:
OK, no problem.